app/main/views/marketplace.py

```diff
@@ -73,3 +73,30 @@ def get_brief_by_id(framework_slug, brief_id):
         brief=brief,
         content=brief_content
     )
+
+
+@main.route('/<framework_slug>/opportunities')
+def list_opportunities(framework_slug):
+    page = process_page(request.args.get('page', default=u"1"))
```

Comment:
Ah nice - the conversion function existed from elsewhere so I just used it, but I'll scrap it from everywhere, should be neater.